### PR TITLE
style(core/styleguide): Adding CSS grids

### PR DESCRIPTION
**Pull request by archana-s**
Added CSS selectors for setting up a page with grids.

Added grids for the skeleton of spinnaker page. Added an example for contents page with filters on the left and inspector to the right. This is just a demo. This should not affect any of the existing styles. This can eventually used while building pages. This currently does not account for responsiveness.

### CODE REVIEW

File: app/scripts/modules/core/src/styleguide/src/styles/grids.css [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+.main-container {
+  position: absolute;
+  left: 0;
+  top: 0;
+  bottom: 0;
+  right: 0;
+
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 42px 80px 68px 1fr;
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/styleguide/src/styles/grids.css]: I don't think we can guarantee the height of the header and nav rows, unfortunately - can these be set to `auto`, leaving the `1fr` for the `contents`?

Also worth considering where the blesk notification would go (just need to make sure it's inside the `secondary-nav`, maybe)

File: app/scripts/modules/core/src/styleguide/src/styles/grids.css [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  overflow-y: auto;
+}
+
+/* View with filter and inspector */
+.content-container.with-filter .filter-area {
+  grid-column-start: 1;
+  grid-column-end: 4;
+}
+
+.content-container.with-inspector .inspector-area{
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/styleguide/src/styles/grids.css]: missing a space before `{`

File: app/scripts/modules/core/src/styleguide/src/styles/grids.css [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  grid-column-start: 10;
+  grid-column-end: 13;
+}
+
+.content-container.with-filter .contents-area{
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/styleguide/src/styles/grids.css]: space before `{`

File: app/scripts/modules/core/src/styleguide/src/styles/grids.css [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+  overflow-y: auto;
+}
+
+/* Colors & borders for marking */
+main > * {
+  padding: 10px;
+}
+
+header {
+  background-color: #3A5469;
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/styleguide/src/styles/grids.css]: should this use a CSS variable declared color? (same question for the rules below)

**Comment by archana-s** [comment]
tested this with library management. Will implement a page or a container with grids and send this diff out with that

**Pull request closed without merging**